code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -36,7 +36,7 @@
 		return
 
 	var/mob/living/carbon/human/H = L
-	if(!H.dna || HAS_TRAIT(H, TRAIT_GENELESS))
+	if(!H.dna || HAS_TRAIT(H, TRAIT_GENELESS) || GODMODE)
```

Comment:
you are checking if GODMODE is true
GODMODE is a define, standing for (1<<4), or 16
you need to check if GODMODE is in the mob's status flags.

Reply:
specifically, you want to replace this with `H.status_flags & GODMODE`